internal/pkg/mount/manager/owned/owned.go

```diff
@@ -14,59 +14,33 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-// MountPointsForDevice returns the mountpoints required to boot the system.
-func MountPointsForDevice(devpath string) (mountpoints *mount.Points, err error) {
+// MountPointsFromLabels returns the mountpoints required to boot the system.
+func MountPointsFromLabels(grow bool) (mountpoints *mount.Points, err error) {
```

Comment:
Not a big fan of the boolean argument. This function is used in a number of places, and I feel like we shouldn't have to specify whether or not the partition should be resized every time its' invoked.

Reply:
It's not super heavy, and there's definitely room to DRY it a bit. 
```internal//app/machined/internal/platform/aws/aws.go
internal//app/machined/internal/platform/azure/azure.go
internal//app/machined/internal/platform/baremetal/baremetal.go
internal//app/machined/internal/platform/googlecloud/googlecloud.go
internal//app/machined/internal/platform/packet/packet.go
internal//app/machined/internal/platform/vmware/vmware.go
internal//pkg/installer/installer.go
internal//pkg/mount/manager/owned/owned.go```

We could always jiggle it with variadic parameters if we want to make clever, but not sure that's really needed.